civihr_default_theme/scss/pages/_hr-details.scss

```diff
@@ -65,3 +72,194 @@
     }
   }
 }
+
+.view-mydetails-personal {
+  .chr_panel--my-details__personal__contact_id span {
+    @include views-label;
+  }
+
+  .pane-block {
+    border-bottom: 1px solid $gray-light;
+    margin-bottom: 0;
+  }
+
+  .views-row {
+    min-height: 80px;
+    padding-left: 100px;
+    position: relative;
+  }
+
+  .views-field-image-URL {
+    left: 0;
+    position: absolute;
+    top: 0;
+
+    img {
+      border-radius: 50%;
```

Comment:
not sure but, is this still about the user profile picture? If no fine, but if it's yes then again we have a fully reusable block already styled, and we shouldn't need this

Reply:
Removed.